Gui/opensim/installer_files/Windows/make_installer.nsi

```diff
@@ -11,7 +11,10 @@
 
   ;Name and file
   Name "OpenSim 4.0 Beta"
-  OutFile "opensim-4.0-win64.exe"
+  OutFile "opensim-4.0Beta-win64.exe"
```

Comment:
For Mac, we are able to use Ant to "configure" a similar file to avoid explicitly listing the version number.

Reply:
Definitely,  where does it get set?

Reply:
https://github.com/opensim-org/opensim-gui/blob/587ee9d90b052d1335f99b41be2455717c6caf90/Gui/opensim/build.xml#L236-L241

https://github.com/opensim-org/opensim-gui/blob/587ee9d90b052d1335f99b41be2455717c6caf90/Installer/macOS/Distribution.xml#L5

Reply:
Yes I saw that but wasn't sure where app.version is set? Now I see it under the Project Properties. Thanks @chrisdembia